### DISCUSSION TITLE
bash: v: command not found

### DISCUSSION THREAD

**Discussion by kadekkurniawan** [vlang/v#15875] [Installation Issues]
when I tried to see the v version using `v -v` in git bash I got this message `bash: v: command not found`, however when I tried the same command (v -v) in the command prompt (cmd.exe) it's showing me the v version.

how to make  v command available in git bash?

**Comment by changrui** [comment]
`./v symlink`

**Reply by JalonSolov** [reply]
Or `sudo ./v symlink` as necessary.

**Reply by kadekkurniawan** [reply]
When the installation process I already run that command, do you mean to run that command again?

**Reply by JalonSolov** [reply]
Did you install V in a Windows shell, or the git bash shell?

**Reply by kadekkurniawan** [reply]
i install it in windows shell

**Reply by JalonSolov** [reply]
I don't use git bash, so you will likely have to experiment a bit.

On Linux, the `v symlink` command creates `/usr/local/bin/v` as a link that points to the real V executable.  You may need to do something like that in git bash to make it work the way you expect.

Truthfully... I recommend getting rid of git bash and all that, and simply enabling WSL2 on your system.  Then you can install a real version of Linux alongside Windows, and have a much better experience.  That's what I did for several years before I got rid of Windows altogether and just went straight Linux.

**Comment by spytheman** [comment]
Try running `set` in your Git bash shell, and look at PATH. 

If it lacks the folder where V is, you can fix that by editing `~/.bashrc` and adding something like:
`export PATH="/path/to/your/vfolder:$PATH"`, then restart your shell and look again with `set`, what PATH contains. If everything is ok, `v` should be available to you now.

**Reply by saksh-2505** [reply]
This one worked for me.  Thanks
